tools/docker/development/README.md

```diff
@@ -0,0 +1,29 @@
+# Docker
+
+Note that this Dockerized environment **is not suitable** for production deployments, see [Installations](https://rathena.github.io/user-guides/installing/) instead or [Docker in Production](https://github.com/rathena/rathena/tree/master/tools/docker/production).
+
+### How to setup :computer:
+
+1. `docker-compose up -d` to spin up dev container and database (ensure port `3306` is free)
+2. `docker exec -it rathena-dev bash` to connect to dev container
+3. All rAthena development commands can be executed inside the dev container, such as compiling (`./configure`, `make clean server`) and starting the server (`./athena-start`, `gdb map-server`, etc ...)
+4. `docker-compose down` outside the dev container when done to close database and free resources
+
+#### Tips & tricks :beginner:
+
+- Ensure you don't have a database running locally and listening on port `3306` this will cause the database container to fail starting up.
```

Comment:
```suggestion
- Ensure you don't have a database running locally and listening on port `3306` as this will cause the database container to fail to start up.
```